src/server.h

```diff
@@ -1312,8 +1312,8 @@ struct redisServer {
     long long stat_expired_time_cap_reached_count; /* Early expire cycle stops.*/
     long long stat_expire_cycle_time_used; /* Cumulative microseconds used. */
     long long stat_evictedkeys;     /* Number of evicted keys (maxmemory) */
-    long long stat_eviction_exceeded_time;  /* Total time over the memory limit */
-    long long stat_last_eviction_exceeded_time; /* Timestamp of current eviction start */
+    monotime stat_total_eviction_exceeded_time;  /* Total time over the memory limit */
```

Comment:
is that really a `monotime` type?  i think just a `long long` with `us` units (better mention the units in the comment)

Reply:
`stat_last_eviction_exceeded_time` is timestamp, so it is `monotime`.
`stat_total_eviction_exceeded_time` can be `unsigned long long`.

Reply:
yes, that's what i meant. and if it's just a long long, then we need to document the units.